src/main/java/org/opendaylight/yangtools/triemap/CNode.java

```diff
@@ -187,7 +186,8 @@ private static int elementSize(final BasicNode elem, final ImmutableTrieMap<?, ?
             BasicNode sub = arr[i];
             if (sub instanceof INode) {
                 final INode<?, ?> in = (INode<?, ?>) sub;
-                final MainNode<?, ?> inodemain = Verify.verifyNotNull(in.gcasRead(ct));
+                final MainNode<?, ?> inodemain = Optional.ofNullable(in.gcasRead(ct))
```

Comment:
Add a replacement for verifyNotNull() in a package-private utility class. Do not bounce through Optional in there.